### PR TITLE
Defines a proper CODEOWNERS file

### PR DESCRIPTION
**Pull request by lukaszlenart**
[About code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

### CODE REVIEW

**Comment by lukaszlenart** [comment]
//cc: @apache/struts-committers

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Request PR review from any Apache Struts committer
+* @apache/struts-committers
```
Comment (kusalk) [inline CODEOWNERS:2]: I think GitHub doesn't like the `*`

Comment (kusalk) [reply CODEOWNERS:2]: Ah it might be complaining because it's a private team

Comment (lukaszlenart) [reply CODEOWNERS:2]: Right... not sure why this team is private as all committers are publicly known

Comment (sepe81) [reply CODEOWNERS:2]: But where exactly are all struts-committers defined as a group on GitHub? I couldn't find any reference.

Comment (lukaszlenart) [reply CODEOWNERS:2]: I assume this is autogenerated group by ASF Infra base on the ASF records, yet I'm not sure which one is used for that as it doesn't include all the committers
https://people.apache.org/phonebook.html?unix=struts

Comment (kusalk) [reply CODEOWNERS:2]: I see a list of 11 people - presumably the Struts committers who have made their Apache org membership public (private by default)?

Comment (lukaszlenart) [reply CODEOWNERS:2]: As far I understand you must link your GH account to the ASF account to be added to the team, yet still there is a difference.

Comment (lukaszlenart) [reply CODEOWNERS:2]: For reference https://issues.apache.org/jira/browse/INFRA-24905

**Pull request merged**